stats/player_general_traditional_total.py

```diff
@@ -43,68 +43,68 @@ def populate_season(self, season_id):
-                'fta_rank': row[46],
-                'ft_pct_rank': row[47],
-                'oreb_rank': row[48],
-                'dreb_rank': row[49],
-                'reb_rank': row[50],
-                'ast_rank': row[51],
-                'tov_rank': row[52],
-                'stl_rank': row[53],
-                'blk_rank': row[54],
-                'blka_rank': row[55],
-                'pf_rank': row[56],
-                'pfd_rank': row[57],
-                'pts_rank': row[58],
-                'plus_minus_rank': row[59],
-                'nba_fantasy_pts_rank': row[60],
-                'dd2_rank': row[61],
-                'td3_rank': row[62],
-                'cfid': row[63],
-                'cfparams': row[64]
+                'team_id': row[3],
+                'age': row[5],
+                'gp': row[6],
+                'w': row[7],
+                'l': row[8],
+                'w_pct': row[9],
+                'min': row[10],
+                'fgm': row[11],
+                'fga': row[12],
+                'fg_pct': row[13],
+                'fg3m': row[14],
+                'fg3a': row[15],
+                'fg3_pct': row[16],
+                'ftm': row[17],
+                'fta': row[18],
+                'ft_pct': row[19],
+                'oreb': row[20],
+                'dreb': row[21],
+                'reb': row[22],
+                'ast': row[23],
+                'tov': row[24],
+                'stl': row[25],
+                'blk': row[26],
+                'blka': row[27],
+                'pf': row[28],
+                'pfd': row[29],
+                'pts': row[30],
+                'plus_minus': row[31],
+                'nba_fantasy_pts': row[32],
+                'dd2': row[33],
+                'td3': row[34],
+                'gp_rank': row[35],
+                'w_rank': row[36],
+                'l_rank': row[37],
+                'w_pct_rank': row[38],
+                'min_rank': row[39],
+                'fgm_rank': row[40],
+                'fga_rank': row[41],
+                'fg_pct_rank': row[42],
+                'fg3m_rank': row[43],
+                'fg3a_rank': row[44],
+                'fg3_pct_rank': row[45],
+                'ftm_rank': row[46],
+                'fta_rank': row[47],
+                'ft_pct_rank': row[48],
+                'oreb_rank': row[49],
+                'dreb_rank': row[50],
+                'reb_rank': row[51],
+                'ast_rank': row[52],
+                'tov_rank': row[53],
+                'stl_rank': row[54],
+                'blk_rank': row[55],
+                'blka_rank': row[56],
+                'pf_rank': row[57],
+                'pfd_rank': row[58],
+                'pts_rank': row[59],
+                'plus_minus_rank': row[60],
+                'nba_fantasy_pts_rank': row[61],
+                'dd2_rank': row[62],
+                'td3_rank': row[63],
+                'cfid': row[64],
+                'cfparams': row[65]
```

Comment:
I agree that indexing by into the column description would be better. This works for now though.

Reply:
I will work on adding that next. Trying to get a sense of the incoming data. I am not able to reproduce this call(http://stats.nba.com/stats/leaguedashplayerstats) from my Rest API client. 

Where do you find the documentation for these NBA APIs?

Reply:
This repo is a great reference: https://github.com/swar/nba_api/tree/master/docs/nba_api/stats/endpoints I _think_ that I credit them in the README or wiki somewhere, hopefully...

Reply:
Thanks. I think you did. I must have missed it.

I am still having trouble though. Not sure why I am not getting a response. The request is going through for sure, but the response just never comes, even 5 minutes in. The client is stuck waiting.

Reply:
Ah yeah, are you using the same headers as this library uses? The api is very finicky. The headers must match up, and you can only request once every half second to avoid rate limiting. That is the main reason this lib takes so long, dancing around the rate limits.

Reply:
I copied them exactly. I use a REST API client called [Insomnia](https://insomnia.rest/products/insomnia). It's kind of like Postman, if you are used that instead.

The headers worked for one call, but not for these leaguedashplayer type endpoints. Very strange.

Reply:
Maybe we can find a substitute endpoint. What did you want to pull from that endpoint specifically, it seems pretty massive.

Reply:
Nothing in particular. This is one endpoint being used in the code, and I wanted to reproduce the call locally to see the column headers.

It's alright though, I can just take a dump from pdb, to get the payload output. I am just slightly annoyed that I can't get it to run on the client.

Reply:
Figured out the issue. I was sending in `Regular+Season`, when it should be `Regular Season`. The client automatically coverts the space to `%`.

Reply:
Ah makes sense, glad it ended up working out.